handbook/handbook.people.onboarding.checklist.md

```diff
@@ -0,0 +1,256 @@
+
+### Continuous Improvement
+
+In the interests of paying it forward and contributing to our core value of [Iteration](https://handbook.dendron.so/notes/b89ba854-72fb-4ebc-a8a0-55960b89e9dc.html#iteration) - we encourage you to suggest changes and updates to this note and the handbook, both of which can be done via Pull Request (PR).
+
+Along with this you will complete onboarding on Day Thirty by being asked to fill out the [Onboarding Satisfaction Survey](TODO), which is an opportunity to share feedback on your general Onboarding Experience.
+
+---
+
+### Before Starting at Dendron
+
+<details>
+<summary>Manager</summary>
+
+1. [ ] Check to see if the new hire's Rippling onboard has been completed, if not send out a reminder.
+1. [ ] Schedule a Zoom video call with the new team member for the start of their first day to welcome them to the team and set expectations. Send them an invitation to their personal email. The team members Dendron email address will only be activated on their first day. You can find their personal email address on Rippling.
+1. [ ] Send an email to the new team member's personal email address welcoming them to Dendron. Provide them with a link to this note to help with a great first day experience.
+1. [ ] Read through Gitlab's [Building Trust Handbook page](https://about.gitlab.com/handbook/people-group/learning-and-development/building-trust/).
+1. [ ] Organize a smooth onboarding plan with clear starting tasks and pathway for new team member (copy this note for the team member as a starting point)
+1. [ ] Review and add any Job-specific tasks to this note. Complete any manager tasks and coordinate with kpats@ if any changes are needed.
+
+### Google account
+
+1.  [ ] Log on to the [Google Admin console](https://admin.google.com/ac/users) and verify the chosen email address is not conflicting with a current account. Then assign the email to team meber and add the email address to [Rippling](https://app.rippling.com/dashboard) in the work email field.
+
+### Accounts and Access
+
+1. [ ] Add the team members to the `Dendron Team` role in Discord. This will give them access to the relevant channels.
+1. [ ] Invite team member to recurring team meetings, consider pressing `don't send`, this will reduce noise in their inbox on day 1.
+1. [ ] Schedule weekly [1:1 meetings](https://about.gitlab.com/handbook/leadership/1-1/) with new team member. Use the first 15 minutes to get to know your new team member on a personal level.
+1. [ ] Set new team member's Github group and project-level permissions as needed.
+</details>
+
+### Day 1 - Getting Started: Accounts and Paperwork
+
+<details>
+<summary>New Team Member</summary>
+
+#### 1Password
+
+1. [ ] :red_circle: Install 1Password. Dendron provides all team members with an account to 1Password. Some teams use 1Password for shared notes and some shared accounts. 1Password is available as a standalone application for macOS, and as a browser extension and CLI tool for Linux.
```

Comment:
do we want to use 1password or rpass? i think rpass already comes with rippling and we already pay for it